tests/constructor/test_config_structure.py

```diff
@@ -0,0 +1,56 @@
+import sys
+sys.path.append('../../')
+
+import hydra
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import *
+
+from hydra.core.config_store import ConfigStore
+from omegaconf import OmegaConf
+
+from src.constructor.config_structure import ConfigParams, Phase
+
+
+@hydra.main(config_path="configs/", config_name="config.yaml")
+def test_config_load_when_full_config_was_define(config):
```

Comment:
was_defined

Reply:
Also, the name of the test suggests a big test, but in fact only two parameters are checked. Shouldn't you check all the parameters that you expect with a bunch of assert statements? (and check the comment below)